packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-link/wallet-link-wrapper.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Text, WalletCard } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import WalletAccount from '../wallet-account/wallet-account';
+import { observer, useStore } from '@deriv/stores';
+import './wallet-link-wrapper.scss';
+
+export type TWalletLinkWrapper = {
+    wallet_details: React.ComponentProps<typeof WalletCard>['wallet'];
+    account_list: {
+        balance: number;
+        currency: string;
+        account_name: string;
+        icon: string;
+    }[];
+};
+
+const WalletLinkWrapper = observer(({ wallet_details, account_list }: TWalletLinkWrapper) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    return (
+        <div className='wallet-link-wrapper'>
+            <div className='wallet-link-wrapper__accounts'>
+                {is_mobile && (
+                    <Text
+                        as='div'
+                        className='wallet-link-wrapper__title-text wallet-link-wrapper__accounts-title'
+                        color='prominent'
+                        size='xxxs'
+                    >
+                        {localize('Your current trading account(s)')}
```

Comment:
could we use `Localize` component here instead of localize

Reply:
Updated.